src/transcription/prerecorded.rs

```diff
@@ -81,11 +84,15 @@ impl<K: AsRef<str>> Transcription<'_, K> {
     /// # Examples
     ///
     /// ```no_run
+    /// # use std::env;
+    /// #
     /// # use deepgram::{
-    /// #     transcription::prerecorded::{Language, Options, UrlSource},
+    /// #     transcription::prerecorded::{
+    /// #         audio_source::UrlSource,
+    /// #         options::{Language, Options},
+    /// #     },
```

Comment:
For discussion: Is it worth exposing these extra module layers to the users?  Might be better to make judicious use of `pub use` here, so the extra layers are only for organization convenience, but the API remains simple.

Reply:
This is something that I have given some thought to. I feel like the only layer worth skipping over would be the `transcription` layer.

Before:
```rust
use deepgram::transcription::prerecorded::{
    audio_source::UrlSource,
    options::{Language, Options},
};
```

After:
```rust
use deepgram::prerecorded::{
    audio_source::UrlSource,
    options::{Language, Options},
};
```

Beyond that I think the organization is useful. In version 0.1.0, I did have everything below the `prerecorded` layer re-exported, but that resulted in [this](https://docs.rs/deepgram/0.1.0/deepgram/prerecorded/index.html). There's a whole bunch of types here, and it's not immediately obvious what each one is used for.